1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,203 @@
+---
+RFC: Update $ErrorView with simplified views for Error Messages
+Author: Jason Helmick
+Status: Draft
+Version: 0.0.0
+Area: PowerShell
+Comments Due: 10/31/2019
+---
+
+# Update $ErrorView with simplified views for Error Messages
+
+When an error occurs in PowerShell, the customers on-screen error message experience currently
+provides a level of detail that obscures the exception message from being recognized and read by
+new or occasional PowerShell users. The addition of a simplified error view will improve both
+comprehension and troubleshooting experience. A new cmdlet 'Resolve-ErrorRecord' will provide
+complete detailed view of the fully qualified error when desired.
+
+## Motivation
+
+The on-screen experience, when receiving an error message,
+is controlled through the views NormalView (the default) and CategoryView. These are user selectable
+through the preference variable $ErrorView.
+This RFC describes Changing '$ErrorView to an enumeration and adding one additional dynamic view
```

Comment:
> This RFC describes Changing `$ErrorView` to an enumeration

You're making a mistake by proposing an enumeration be used for `$ErrorView`, because you're cutting off anyone who wants to build custom error views that way. Also, by using an enumeration, you're forcing yourself to stick with the `View` suffix to maintain backward compatibility with `CategoryView`. You shouldn't do either. Leave it as a string so that anyone who wants to customize their error message views can do so without having to submit a PR and go through a review and approval process. Drop the `View` suffix because it's redundant, and by sticking with `$ErrorView` as a string, you can drop that suffix without breaking backward compatibility. When the value is set, if the incoming value is `CategoryView`, drop the `View` suffix automatically. When the value is set, if it doesn't match a view, or if it is `$null`, fall back to `Normal` (the default). This allows you to remove the redundant suffix, maintain backwards compatibility, _and_ continue offering support to users who want to customize their error view.

Reply:
Changes to the error view, including the ConciseView or as an additional view, is welcomed as a PR.